src/parseUtils.js

```diff
@@ -45,7 +45,7 @@ function parseBundle(bundlePath) {
         // As function name may be changed with `output.jsonpFunction` option we can't rely on it's default name.
         if (
           node.callee.type === 'Identifier' &&
-          args.length === 2 &&
+          (args.length === 2 || args.length === 3) &&
           isArgumentContainsChunkIds(args[0]) &&
           isArgumentArrayConcatContainingChunks(args[1])
```

Comment:
Mm yeah, I can do that. That sounds like a good protection idea.

Do you want me to add the same protection to the plain old `webpackJsonp([<chunks>], <modules>, ...)` call [is before this if-statement](https://github.com/valscion/webpack-bundle-analyzer/blob/7be38bf73fd465859a96464a91e6b9c24dc9ff2b/src/parseUtils.js#L34)?

I could do that in another PR, though, to make this PR more focused :)